### PR TITLE
Update version-script

### PR DESCRIPTION
**Pull request by bwikbs**
Related PR :https://github.com/flutter-tizen/engine/pull/85

Minor changes to achieve the below:
```
{
  global:
    FlutterDesktop*;
  local:
    *;
}
```

### CODE REVIEW

**Comment by bwikbs** [comment]
It's tested on webview example at FHUB

**Comment by swift-kim** [comment]
> Minor changes to achieve the below:

I was talking about the **embedder** binary (`libflutter_tizen.so`) when I left that comment. I guess `FlutterDesktop*;` is not necessary for the engine binary.

**Comment by swift-kim** [comment]
May I ask any reason for this change?

**Comment by bwikbs** [comment]
> May I ask any reason for this change?

Basically, main purpose of this is to prevent unnecessary symbols to show outside.(like FT?)
It's not essential but someone's favorite part :smile:

File: shell/platform/embedder/flutter_engine_exports.lst [hunk 1]
```diff
@@ -8,7 +8,10 @@
 
 {
   global:
-    *;
+    FlutterDesktop*;
+    FlutterEngine*;
+    FlutterPlatformMessage*;
+    *Dart*;
```
Comment (swift-kim) [inline shell/platform/embedder/flutter_engine_exports.lst:14]: How did you get this list?

Also you need to update my comment in line 5 to reflect this change.

**Review by swift-kim — COMMENTED** [review]
Actually I'm against this change because it makes another non-trivial diff between the upstream engine and our engine, and can potentially cause symbol resolution failures.

**Comment by bwikbs** [comment]
> Actually I'm against this change because it makes another non-trivial diff between the upstream engine and our engine, and can potentially cause symbol resolution failures.

Hmm... I'm hard to understand .. I don’t think it’s an important issue that needs discussion.
I'll listen to other's opinion for 1~2 day and withdraw this PR if there is nothing else.

File: shell/platform/embedder/flutter_engine_exports.lst (hunk 1 above)
Comment (bwikbs) [reply shell/platform/embedder/flutter_engine_exports.lst:14]: > How did you get this list?

I added it after seeing the error message during build & running.

**Pull request closed without merging**